app-nuxt-starter/pages/index.vue

```diff
@@ -1,15 +1,10 @@
 <script setup>
-import { navigateTo, useFetch, useRoute } from '#app'
```

Comment:
now pages don't need to deal with authentication as well. it's done by the server middleware.